packages/network-enablement-controller/src/NetworkEnablementController.test.ts

```diff
@@ -329,47 +361,38 @@ describe('NetworkEnablementController', () => {
 
   describe('init', () => {
     it('initializes network enablement state from controller configurations', () => {
-      const { controller } = setupController();
-
-      jest
-        // eslint-disable-next-line dot-notation
-        .spyOn(controller['messagingSystem'], 'call')
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        .mockImplementation((actionType: string, ..._args: any[]): any => {
-          // eslint-disable-next-line jest/no-conditional-in-test
-          if (actionType === 'NetworkController:getState') {
-            return {
-              selectedNetworkClientId: 'mainnet',
-              networkConfigurationsByChainId: {
-                '0x1': { chainId: '0x1', name: 'Ethereum Mainnet' },
-                '0xe708': { chainId: '0xe708', name: 'Linea Mainnet' },
-                '0x2105': { chainId: '0x2105', name: 'Base Mainnet' },
-              },
-              networksMetadata: {},
-            };
-          }
-          // eslint-disable-next-line jest/no-conditional-in-test
-          if (actionType === 'MultichainNetworkController:getState') {
-            return {
-              multichainNetworkConfigurationsByChainId: {
-                'eip155:1': { chainId: 'eip155:1', name: 'Ethereum Mainnet' },
-                'eip155:59144': {
-                  chainId: 'eip155:59144',
-                  name: 'Linea Mainnet',
-                },
-                'eip155:8453': { chainId: 'eip155:8453', name: 'Base Mainnet' },
-                'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp': {
-                  chainId: 'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp',
-                  name: 'Solana Mainnet',
-                },
-              },
-              selectedMultichainNetworkChainId: 'eip155:1',
-              isEvmSelected: true,
-              networksWithTransactionActivity: {},
-            };
-          }
-          throw new Error(`Unexpected action type: ${actionType}`);
-        });
+      const mockGetNetworkState = jest.fn().mockImplementation(() => ({
```

Comment:
I know we ought to migrate these tests so we aren't mocking the messenger, but could we leave these changes to another PR or do we have to do it here?